packages/service-registry-api/src/primitives.rs

```diff
@@ -62,3 +62,21 @@ pub enum AuthorizationState {
     Authorized,
     Jailed,
 }
+
+// Represents any modifiable fields of the Service struct
+// Any non-None field overwrites the value currently stored in the Service object
+#[cw_serde]
+pub struct UpdatedServiceParams {
+    pub min_num_verifiers: Option<u16>,
+    pub max_num_verifiers: Option<Option<u16>>,
+    pub min_verifier_bond: Option<nonempty::Uint128>,
+    pub unbonding_period_days: Option<u16>,
+}
```

Comment:
this doesn't feel like it should be in primitives, it's pretty specific to the api